### PR TITLE
Publish minutes for 2016-11-03 security meeting

### PR DESCRIPTION
**Pull request by seabaylea**
@swift-server/stakeholders Minutes from today's Security meeting.

### CODE REVIEW

**Comment by adellibovi** [comment]
@seabaylea that was fast! :) I guess we will review it here instead of the online document.
A question about the "process" diagram, where are we going to reflect this? Should we wait for tomorrow meeting in order to upload somewhere in our repo?

**Comment by seabaylea** [comment]
Yeah, I thought doing the review via a PR might make more sense. I'll probably push the process diagram into the root of the project - it should probably be in the readme.md somewhere.

File: meetings/security/2016-11-03/minutes.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Minutes:
+_Q) What are our intended timelines:_  
+* We really need to have a ”1.0” for Swift 4. Previews should/will be available way before that though.
+* We’re not constrained by Swift 4.0 though since we can release as packages.
+
+_Q) How important do we see that security is on the server side? Couldn’t you just put a reverse proxy in front?_  
+* You can if you put the proxy in the same container, so HTTPS isn’t/shouldn’t be a show stopper…  
+* More important for outbound, as its needed for HTTPS requests of other services
+* However, code portability is a must. We want the API layer to be shared among different platforms - including iOS clients and Linux/macOS servers. The implementation doesn’t have the be the same in all cases, but the APIs should be (if at all possible).
```
Comment (adellibovi) [inline meetings/security/2016-11-03/minutes.md]: Small typo: have the be the same. have to be the same

File: meetings/security/2016-11-03/minutes.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+_Q) Should be build in Swift vs reuse existing tech?_  
+* Ideally want to reuse existing components, so we don’t have to maintain complex code, deal with CVEs, etc. There’s no desire to re-invent the wheel.
+
+_Q) If we’re reusing tech, how low a level do we start the API at? Should it be just a layer on, say, common crypto?_  
+* The starting point is probably just to build a common “translation layer” in Swift, and then see if we want to build abstraction layers on top of that.  
+* It’s easier to build up than it is to build down.
+
+_Q) Which tech would we use?_  
+* If we want to be consistent on iOS clients and servers, then we need to use CommonCrypto/SecureTransport/Keychain at least there, and it would therefore make sense to on macOs as well.
```
Comment (adellibovi) [inline meetings/security/2016-11-03/minutes.md]: macOS

**Review by adellibovi — COMMENTED** [review]
I just commented a small typo. But the document reflects nicely what we spoke about today. LGTM 👍

**Comment by seabaylea** [comment]
Breaking the rule and self-merging as its just meeting minutes and it has been reviewed.

**Pull request merged**